app/controllers/users_controller.rb

```diff
@@ -1,3 +1,23 @@
 class UsersController < ApplicationController
-  def new; end
+  def new
+    @user = User.new
+  end
+
+  def show
+    @user = User.find_by id: params[:id]
```

Comment:
xử lý thêm trường hợp @user là nil nữa e nha, vd:
```suggestion
    @user = User.find_by id: params[:id]
    
    unless @user
      flash ...
      redirect_to ...
    end
```